File: backend/components/metiers/administration_publique/utils/exporter/helpers/exporter_helper.test.js
// exporter_helper.test.js
// Tests unitaires JS pour exporter_helper
const { formatExportJSON } = require('./exporter_helper');

describe('formatExportJSON', () => {
  it('formate un objet en JSON compacté', () => {
    const obj = { a: 1, b: 'x' };
    expect(formatExportJSON(obj)).toBe('{"a":1,"b":"x"}');
  });
});

File: backend/components/metiers/administration_publique/utils/exporter/helpers/exporter_helper.test.py
# exporter_helper.test.py
# Tests unitaires Python pour exporter_helper
from .exporter_helper import format_export_json

def test_format_export_json():
    obj = {'a': 1, 'b': 'x'}
    assert format_export_json(obj) == '{"a": 1, "b": "x"}' or format_export_json(obj) == '{"b": "x", "a": 1}'
